packages/atomic/src/components/search/generated-answer/atomic-generated-answer/atomic-generated-answer.pcss

```diff
@@ -17,6 +32,16 @@
   background: var(--atomic-primary-background);
 }
 
+atomic-modal {
```

Comment:
IIRC the modal is injected outside of this component's shadow dom, no? Does this CSS actually have any effect?